code/modules/spells/roguetown/martialartist.dm

```diff
@@ -0,0 +1,212 @@
+// LORE : The Martial Artist has perfected a beautiful union of the arcyne with martial arts, and under Noc's gaze, he teaches the weak to defend themselves. 
+
+// some defines to make things work
+
+
+/mob/living/carbon/get_bodypart(zone)
+	if(!zone)
+		zone = BODY_ZONE_CHEST
+	for(var/obj/item/bodypart/bodypart as anything in bodyparts)
+		if(bodypart.body_zone == zone)
+			return bodypart
+		for(var/subzone in bodypart.subtargets)
+			if(subzone != zone)
+				continue
+			return bodypart
```

Comment:
Get this out of here. This is just copypasted from the definition in `code/modules/surgery/bodyparts/helpers.dm`. It shouldn't be here.